static/app/components/badge/groupPriority.tsx

```diff
@@ -211,6 +211,7 @@ export function GroupPriorityDropdown({
     () => makeGroupPriorityDropdownOptions({onChange}),
     [onChange]
   );
+  const [isDropdownOpen, setIsDropdownOpen] = useState(false);
```

Comment:
`DropdownMenu`'s `trigger` prop supplies the open state as the second argument in the render function for cases like this:

<img width="503" alt="image" src="https://github.com/getsentry/sentry/assets/44172267/90787c91-cc90-4559-98a4-525a4cef9d8c">